ruby/alias_manager.rb

```diff
@@ -0,0 +1,51 @@
+# for vowels/consonants - compare letter to v/c index
+# 	return index+1
+# 	edges - u, z , spaces
+
+def jumble(name)
+	vowels = "aeiou"
+	consonants = "bcdfghjklmnpqrstvwxyz"
+	jumbled_name = ""
+	#for each letter in name (lower), return index+1 to jumbled (watch edges)
+	name.downcase.chars.each do |letter|
+		jumbled_name<<" " if letter == " "
+		if vowels.include?(letter)

```

Comment:
We could separate this long set of conditionals into or or even 2 more methods.

consider something like

```
def set_letter(letter)
  vowels = "aeiou"

  return letter if letter == " "

  if vowels.include?(letter)
    set_vowels(letter)
  else
    set_consonants(letter)
  end

  letter
end

def set_vowels(letter)
  vowels = "aeiou"

  # code
end

def set_consonants(letter)
  consonants = "bcdfghjklmnpqrstvwxyz"

  # code
end
```